src/plugins/vuetify.js

```diff
@@ -34,6 +34,8 @@ const cuttleTheme = {
     'neutral-lighten3': '#F2F4F5',
     'neutral-lighten4': '#FAFAFA',
     'neutral-lighten5': '#FAFBFC',
+    'surface-1': '#4A2416',
```

Comment:
i.e. chocolate

Reply:
The names here aren't very descriptive, but that's something we can mess with later. I think we should hit up some of the designer folks and get a naming structure for our colors. Maybe work up a panel and have something like:
- `background-primary`
- `background-secondary`
- `button-primary`
- `button-secondary`

There are more, maybe @peterancourt would want to weigh in. Adding these to Storybook seems like a no-brainer.

Reply:
Agreed both on adding to storybook and redefining the whole color system. Honestly I think it will likely make sense to get rid of the purple and orange in the theme altogether and make the chocolate and cream colors the primary and secondary, but we should get some design input before making the change

Reply:
https://github.com/cuttle-cards/cuttle/issues/397